src/auto_pickup.h

```diff
@@ -38,7 +38,7 @@ class auto_pickup
                 bool bExclude;
 
                 cRules() {
-                    this->sRule = "";
+                    this->sRule.clear();
```

Comment:
This line is completely pointless. The member `sRule` is default constructed, and that means it *is* already empty.